pqarrow/arrowutils/sort_test.go

```diff
@@ -206,6 +207,47 @@ func TestSortRecord(t *testing.T) {
 	}
 }
 
+func TestSort_Binary(t *testing.T) {
```

Comment:
Wouldn't it be more ergonomic to add this to the existing `TestSortRecord` test?

Reply:
@asubiotto 

Not possible for now. With the record builder , we map `string` to `*array.String`. There is no `binary` type in the `schema` proto yet so we can't build `*array.Binary` columns or  dictionary of `*array.Binary`

The reason we are discussing `v1alpha1` and `v1alpha2` is I wanted to introduce `binary` type .

This PR came without tests because it is a supplementary PR.  I didn't mind your test recommendation thats why I didn't bring this up.

Personally I would prefer we omit the test here, so I can make sure we can represent and build `*array.Binary` column. The test coverage for this case will be on that work  by extending `TestSortRecord` test case matrix. By which point I will again have to delete `TestSort_Binary`

I'm fine with whatever option you pick to be happy with this PR .

Reply:
ok, in that case let's omit.